common/djangoapps/student/management/commands/set_superuser.py

```diff
@@ -0,0 +1,47 @@
+from optparse import make_option
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+import re
+
+
+class Command(BaseCommand):
+    option_list = BaseCommand.option_list + (
+        make_option('--unset',
+                    action='store_true',
+                    dest='unset',
+                    default=False,
+                    help='Set is_superuser to False instead of True'),
+    )
+
+    args = '<user|email> [user|email ...]>'
+    help = """
+    This command will set is_superuser to true for one or more users.
+    Lookup by username or email address, assumes usernames
+    do not look like email addresses.
+    """
+
+    def handle(self, *args, **options):
+        if len(args) < 1:
+            raise CommandError('Usage is set_superuser {0}'.format(self.args))
+
+        for user in args:
```

Comment:
Yeah I think this would be pretty easy to do. Wrap the User.objects.get calls in a Try, and output a message if there's an exception (eg `print "Could not get user with identifier {}".format(user)`)